### DISCUSSION TITLE
Deleting a Record using SDK does not work

### DISCUSSION THREAD

**Discussion by AnthonyDelaCruz** [supabase/supabase#11684] [Questions] [answered]
I am trying to do a simple deletion of a record using the SDK's `.delete()` method with the columns id (type `uuid`). But when I try to run it, it throws this error. `"message": "invalid input syntax for type bigint"`

**Comment by GaryAustin1** [comment, marked as answer]
Did you redo the column or table at some point?

Try restarting the database in the general settings.

**Reply by AnthonyDelaCruz** [reply]
Yes I did recreate the table. I'll try out what you suggested.

**Reply by AnthonyDelaCruz** [reply]
Restarting the database worked 👍 Thanks for the suggestion!

**Reply by GaryAustin1** [reply]
I generated an issue...   https://github.com/supabase/supabase/issues/11714
